src/lsdb/dask/merge_catalog_functions.py

```diff
@@ -273,3 +291,14 @@ def align_catalog_to_partitions(
     )
     partitions = get_partition(pixels)
     return list(partitions)
+
+
+def create_no_coverage_crossmatch_df(
+    left_df: pd.DataFrame, suffix: str, meta_df: pd.DataFrame
+) -> pd.DataFrame:
+    """Creates the crossmatch DataFrame for partitions in the left catalog that have no coverage
+    in the right catalog."""
+    no_coverage_df = meta_df.copy(deep=False)
+    for name, _ in left_df.dtypes.items():
+        no_coverage_df[name + suffix] = left_df[name]
```

Comment:
```suggestion
    no_coverage_df = pd.DataFrame(data=np.full((len(left_df), len(meta_df.columns)), fill_value=None), columns=meta_df.columns, index=left_df.index)
    no_coverage_df[left_df.columns] = left_df
```

Does this work?

Reply:
Yes, I just needed to add the suffixes to left columns. Thanks!